mcp-go/SSE_USAGE.md

```
@@ -0,0 +1,377 @@
+curl http://localhost:8081/sse &
+curl http://localhost:8082/sse &
+```
+
+## Container Deployment
+
+### Docker Example
+
+```dockerfile
+FROM golang:1.23-alpine AS builder
+WORKDIR /build
+COPY . .
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)